src/cogent3/app/sample.py

```diff
@@ -246,21 +256,63 @@ def __init__(self, allowed_frac=0.99, motif_length=1, moltype=None):
             sequences split into non-overlapping tuples of this size.
         moltype : str
             molecular type, must be either DNA or RNA
+
+        Examples
+        --------
+
+        Create a sample alignment and an app that excludes highly gapped sites.
+        Sites with over 99% gaps are excluded by default
+
+        >>> from cogent3 import make_aligned_seqs, get_app
+        >>> aln = make_aligned_seqs({"s1": "ACGA-GA-CG", "s2": "GATGATG-AT"})
+
+        >>> app = get_app("omit_gap_pos", moltype="dna")
+        >>> result = app(aln)
+        >>> print(result.to_pretty(name_order=["s1", "s2"]))
+        s1    ACGA-GACG
+        s2    GATGATGAT
+
+        Create an app that excludes all aligned sites with over 49% gaps
+
+        >>> app = get_app("omit_gap_pos", allowed_frac=0.49, moltype="dna")
+        >>> result = app(aln)
+        >>> print(result.to_pretty(name_order=["s1", "s2"]))
+        s1    ACGAGACG
+        s2    GATGTGAT
+
+        Create an app that excludes non-overlapping alignment columns of three
+        sites that have over 17% gaps. Trailing columns that are less than
+        three sites long are excluded
```

Comment:
as above wrt trailing full stop.

per @GavinHuttley

We need the user to know what motif actually means.  Let's use a biological example.  Replace the above text with.

"To eliminate any codon columns (where a column is a triple of nucleotides) that contain a gap character, we use the motif_length argument."

Replace code example with ...
```
>>> app = get_app("omit_gap_pos", allowed_frac=0, motif_length=3, moltype="dna")
>>> result = app(aln)
>>> print(result.to_pretty(name_order=["s1", "s2"]))
s1    ACG
s2    GAT
```